packages/node-bridge/test/bridge.test.js

```diff
@@ -112,7 +112,7 @@ test('[APIGatewayProxyEvent] cookie handling', async () => {
   const body = JSON.parse(Buffer.from(result.body, 'base64').toString());
   expect(body.method).toBe('POST');
   expect(body.path).toBe('/');
-  expect(body.headers.cookie).toBe('cookie-1, cookie-2, cookie-3');
+  expect(body.headers.cookie).toBe('cookie-1; cookie-2; cookie-3');
```

Comment:
Must be also changed here: https://github.com/dealmore/terraform-aws-next-js/blob/10f75175746f07d1c0e08bea3f5461c6adc0fea3/packages/node-bridge/test/bridge.test.js#L94

(Otherwise tests will fail)

Reply:
Should be
```js
cookies: ['cookie-1; cookie-2', 'cookie-3'], 
```

(Thanks to GitHub I cannot make a suggestions on unchanged lines 🙃)